cpp/oneapi/dal/algo/pca/backend/cpu/train_kernel_cov.cpp

```diff
@@ -27,7 +27,7 @@ using dal::backend::context_cpu;
 
```

Comment:
Is it a good Idea to commit both clang format specification and results of it's applying to code?